improver_tests/psychrometric_calculations/wet_bulb_temperature/test_WetBulbTemperature.py

```diff
@@ -214,11 +233,11 @@ def test_too_many_cubes(self):
         with self.assertRaisesRegex(ValueError, msg):
             WetBulbTemperature().process(CubeList([temp, humid, pressure, temp]))
 
-    def test_empty_cube_list(self):
-        """Tests that an error is raised if there is an empty list."""
+    def test_too_few_cubes(self):
```

Comment:
no cube error handling by as_cubelist function, not this class (i.e. this checks extraction logic of this cubelist by this class).